server/app/views/admin/AdminLayout.java

```diff
@@ -84,8 +85,11 @@ private NavTag renderNavBar() {
     String logoutLink = org.pac4j.play.routes.LogoutController.logout().url();
 
     DivTag headerIcon =
-        div(span("C"), span("F").withClasses("font-thin"))
-            .withClasses(AdminStyles.ADMIN_NAV_BAR_LOGO);
+        div(
+            a().withHref(controllers.routes.HomeController.index().url())
```

Comment:
Can you double check on the issue? I would think you want to redirect to `AdminProgramController` and not `HomeController`.

Reply:
Hi Gwen, Calling the AdminProgramController always returns a 403 error. It is the homecontroller which calls the other controllers based on the profile- https://github.com/civiform/civiform/blob/95caf4c71e2a2488039e2b01342609bd781ee10b/server/app/controllers/HomeController.java#L51
public CompletionStage<Result> index(Http.Request request) {
    Optional<CiviFormProfile> maybeProfile = profileUtils.currentUserProfile(request);

    if (maybeProfile.isEmpty()) {
      return CompletableFuture.completedFuture(
          redirect(controllers.routes.HomeController.loginForm(Optional.empty())));
    }

    CiviFormProfile profile = maybeProfile.get();

    if (profile.isCiviFormAdmin()) {
      return CompletableFuture.completedFuture(
          redirect(controllers.admin.routes.AdminProgramController.index()));
    } else if (profile.isProgramAdmin()) {
      return CompletableFuture.completedFuture(
          redirect(controllers.admin.routes.ProgramAdminController.index()));
    } else {